src/pages/sign-up/SignUp.tsx

```diff
@@ -3,9 +3,10 @@ import { useForm, SubmitHandler } from 'react-hook-form'
 import * as Yup from 'yup'
 import { yupResolver } from '@hookform/resolvers/yup'
 import { AiFillEyeInvisible, AiFillEye } from 'react-icons/ai'
-import { useNavigate } from 'react-router-dom'
+import { useNavigate } from 'react-router'
```

Comment:
Normally you should never import anything directly from react-router package. react-router-dom is enough for a browser based app.